wiki/Tournaments/OMF/WC/en.md

```diff
@@ -0,0 +1,485 @@
+  - [Loos - LOVE * BEAM (james039) \[Insane\]](https://osu.ppy.sh/beatmapsets/4970#osu/25466)
+- HardRock
+  - [Kagamine Rin - Tokyo Teddy Bear (NatsumeRin) \[Rin\]](https://osu.ppy.sh/beatmapsets/39838#osu/127138)
+  - [Miki Sayaka vs. Miki Sayaka (fw. Miki Sayaka) - squartatrice (Muya) \[Another\]](https://osu.ppy.sh/beatmapsets/40368#osu/131009)
+- Doubletime HardRock
+  - [Wizet. Nexon - Coke Town (Metroid) \[Hard\]](https://osu.ppy.sh/beatmapsets/5424#osu/26660)
+- DoubleTime
+  - [Megpoid GUMI - Mozaik Role (NatsumeRin) \[PoLKs\]](https://osu.ppy.sh/beatmapsets/20817#osu/74779)
+  - [iconoclasm - perditus paradisus (Sharp) \[Hyper\]](https://osu.ppy.sh/beatmapsets/22212#osu/83343)
+  - [Comp - Gensou no Satellite (Shinxyn) \[Insane (OMF Edit)\]](https://osu.ppy.sh/beatmapsets/4404#osu/63604)
+- Easy
+  - [Hatsune Miku - Toosenbo (NatsumeRin) \[Rin\]](https://osu.ppy.sh/beatmapsets/18266#osu/65429)
+  - [Mame - Aka ni Somaru Dress (Mystica) \[Lolita\]](https://osu.ppy.sh/beatmapsets/33217#osu/108387)
+- Flashlight
+  - [BRIGHT - Ichinen Nikagetsu Hatsuka (Nymph) \[Hard\]](https://osu.ppy.sh/beatmapsets/50270#osu/154582)
+  - [Santana feat. Rob Thomas - Smooth (lesjuh) \[Hard\]](https://osu.ppy.sh/beatmapsets/9781#osu/38695)
+- Tiebreaker
+  - [Ayane - Endless Tears... (lkp) \[Plus\]](https://osu.ppy.sh/beatmapsets/24960#osu/85355)
+
+### Qualifiers
+
+- NoMod
+  - [Susumu Hirasawa - The Secret of The Flowers of Phenomenon (jesus1412) \[Insane\]](https://osu.ppy.sh/beatmapsets/70247#osu/201974)
+  - [O-Life Japan - Yakujinsama no Couple Dance (AngelHoney) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/16990#osu/95954)
+  - [Chata - Remind (Patchouli) \[Overdrive!!\]](https://osu.ppy.sh/beatmapsets/37292#osu/120030)
+- Hidden
+  - [Horie Yui - Coloring (IchiGUN) \[Color of Heart\]](https://osu.ppy.sh/beatmapsets/44433#osu/166749)
+  - [Ruzarin Kashiwagi - Theme of Love (Short Ver.) (Hitoshirenu Shourai) \[Insane\]](https://osu.ppy.sh/beatmapsets/3892#osu/22828)
+- HardRock
+  - [Kola Kid - can't hide your love (Kert) \[Can't\]](https://osu.ppy.sh/beatmapsets/39732#osu/126446)
+- Doubletime HardRock
+  - [Angela Chang - Adventure of True Love (James) \[Hard\]](https://osu.ppy.sh/beatmapsets/3301#osu/21428)
+- DoubleTime
+  - [Mina - Kiss Kiss (osu_CN_yaya) \[Another\]](https://osu.ppy.sh/beatmapsets/9658#osu/38359)
+  - [Midnight Project - Crystal Cinderella (C R E A M) \[Cookietic\]](https://osu.ppy.sh/beatmapsets/28017#osu/96708)
+- Easy
+  - [An - AxiVid (Nakagawa-Kanon) \[Another\]](https://osu.ppy.sh/beatmapsets/66000#osu/192802)
+- Flashlight
+  - [Pendulum - The Vulture (La Cataline) \[Insane\]](https://osu.ppy.sh/beatmapsets/24163#osu/82249)
+
+## Match results
+
+
+### Round of 32
+
+The stats of the Round of 32 can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1Kj0Y1ycdQjPPGIBNUvJqnEi4U5UlBvJPhkXQbsmTYLg/edit?rm=minimal "Google spreadsheet").
```

Comment:
Is this intended? Currently links to a OWC2020 spreadsheet?

Reply:
I failed my copypaste it seems, thanks you for noticing